assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -238,6 +265,28 @@ function SizePositionPanel({
           </Button>
         </Row>
       )}
+      {/** X/Y */}
+      <Row expand>
+        <BoxedNumeric
+          suffix={_x('X', 'Position on X axis', 'web-stories')}
+          value={x}
+          min={MIN_MAX.X.MIN - width}
+          max={MIN_MAX.X.MAX}
+          onChange={(value) => pushUpdate({ x: value })}
```

Comment:
Would be great if the element would actually move on the Page without having to blur first, especially when using keyboard only. That seems to be a general issue though, maybe we could have a debounced callback in the background saving the pending updates. For a future PR though.